server/images.go

```diff
@@ -200,6 +206,11 @@ func CreateModel(name string, path string, fn func(resp api.ProgressResponse)) e
 		return err
 	}
 
+	config := ConfigV2{
+		Architecture: "amd64",
```

Comment:
I'm not 100% sure if we need these or not. I had only stuck them in when I was building up the config structure because that's what Docker was doing. It may work just fine w/o them.

Reply:
I'm going to leave them in to be safe.